### ISSUE TITLE
Publish to Maven Central using Eclipse Tycho project GPG key

### ISSUE THREAD

**Issue by akurtakov** [eclipse/tycho#500]
Wikitext is published to Maven Central using project GPG key as shown (here) [https://ci.eclipse.org/docs/job/mylyn-wikitext-release] . Tycho should do the same so artifacts are signed with the project key instead of developer key and to allow easier switching of release person which now requires everyone that have his key accepted as developer of the given artifacts.

**Comment by akurtakov** [comment]
@laeubi @mickaelistria Any reason you can think to not go that path?

**Comment by mickaelistria** [comment]
At the moment, the Tycho release process consists of a bunch of manual steps performed locally. Some of those steps could go to CI and use project key. But it would also require creating a dedicated CI job I believe. So it's a bit of work.

**Comment by laeubi** [comment]
I think automatic staging would be cool, if we could release more easy and more often that would be beneficial. But how is the project key accessed? Will this be stored at the jenkins?

**Comment by mickaelistria** [comment]
> Will this be stored at the jenkins?

Yes, see https://wiki.eclipse.org/Jenkins#How_can_artifacts_be_deployed_to_OSSRH_.2F_Maven_Central.3F for a hint on how to use it, and https://bugs.eclipse.org/bugs/show_bug.cgi?id=576895 and related bugs/Gerrit for how this was enabled for Eclipse Platform.
